package.json

```diff
@@ -7,7 +7,9 @@
     "node": ">=16"
   },
   "resolutions": {
-    "**/@types/node": "18"
+    "**/@types/node": "18",
+    "**/nan": "2.18.0",
+    "**/cpu-features": "0.0.9"
```

Comment:
`nan` is pulled in transitively via several dependencies: `cpu-features`, `drivelist`, `find-git-repositories`, `node-pty`, and `ssh2`. They all reference `nan` as `nan@^2.14.0`, `nan@^2.17.0`, or `nan@^2.18.0`. Therefore, a `yarn upgrade` or a fresh `yarn.lock` would update it to 2.19.0+.

`cpu-features` is included via `ssh2` as `"~0.0.9"`. So this range also allows bumping the package to a later version. 

As far as I understand, using the resolution field is the only way to prevent pulling in newer versions, since the problematic version ranges are all transient.

Reply:
I see, yeah, that makes sense. Otherwise they'll get hoisted...